docs/resources/codearts_deploy_group.md

```diff
@@ -135,7 +130,7 @@ resource "huaweicloud_codearts_deploy_group" "test" {
   
   lifecycle {
     ignore_changes = [
-      is_proxy_mode,
+      project_id,
```

Comment:
can we use `<project_id>/<group_id>` to import the resource as the project_id is a required parameter.

Reply:
OK